modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,39 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$all_values_string = '';
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		// function_exists will return false for functions disabled with the disable_functions ini directive
+		if ( function_exists( 'print_r' ) ) {
```

Comment:
Hello, @jeherve 

It is me who thanks for collaborating with the community of this incredible and essential plug-in. After investigating further, I discovered that, in Locaweb, it is possible to create a php.ini file in site root folder and override the default "disabled functions" configuration.

So I was able to solve the problem with my site. But what if it was someone without knowledge about? That's why we still need to avoid this warning, but we can work with a simpler solution.

To try to aggregate all the great tips from you and @georgestephanis, I've kept the ```function_exists``` but I'm using it in a more expressive way now. And now, if ```var_export``` is also disabled - which would be *extremely rare* - I left the printing of the variables in charge of the filter ```grunion_print_all_values```.